ci-operator/step-registry/hypershift/mce/agent/info/OWNERS

```diff
@@ -0,0 +1,21 @@
+approvers:
```

Comment:
I think @CrystalChun and @carbonin should be part of the reviewers/approvers. WDYT?